actor/src/main/scala/org/apache/pekko/actor/LightArrayRevolverScheduler.scala

```diff
@@ -105,12 +105,14 @@ class LightArrayRevolverScheduler(config: Config, log: LoggingAdapter, threadFac
 
   override def scheduleWithFixedDelay(initialDelay: FiniteDuration, delay: FiniteDuration)(runnable: Runnable)(
       implicit executor: ExecutionContext): Cancellable = {
+    checkPeriod(delay)
```

Comment:
I think `checkPeriod` is clarify than `checkDelay`, delay may be misunderstand to initialDelay.

Reply:
![Screenshot_20240118_053308_GitHub.jpg](https://github.com/apache/incubator-pekko/assets/501740/26d541b9-0e0b-465b-988d-72c15e31c2cb)

How about test with the length field directly？

Reply:
done.